recipes/nmos-cpp/all/conanfile.py

```diff
@@ -65,7 +67,7 @@ def requirements(self):
 
     def build_requirements(self):
         # nmos-cpp needs CMake 3.17 or higher but CCI doesn't allow version ranges
```

Comment:
```suggestion
```

This comment is not relevant anymore if you use version range.

Reply:
Done.